content/docs/command-reference/exp/push.md

```diff
@@ -47,6 +47,11 @@ to see experiments in the remote.
 This command will also try to `dvc push` all <abbr>cached</abbr> data associated
 with the experiments to [remote storage], unless `--no-cache` is used.
 
+If `studio.token` config or `DVC_STUDIO_TOKEN` environment variable is set, DVC
+will notify Studio about new experiments, and display a Studio project URL to
+view experiments. Check
+[this guide on how to setup an access token](/doc/studio/user-guide/projects-and-experiments/live-metrics-and-plots#set-up-an-access-token).
```

Comment:
```suggestion
view experiments. [Get the token](https://studio.iterative.ai/user/_/profile?section=accessToken) or check
[this guide on how to create and manage an access token](/doc/studio/user-guide/projects-and-experiments/live-metrics-and-plots#create-and-manage-access-token).
```

Rather go through more docs, I would suggest we first try to send people directly to a link to get their token. If they have an on-prem setup or special needs, we can also link to the guide. I'd rather link to the `Create and manage access token` of the guide since it gets directly to how to get the token.

Reply:
Should we mention the Studio URL config also here? Ideally, it should be in the linked guide IMO.

Reply:
Is that underscore URL stable and guaranteed to work? And not the side effect of frameworks that they use?

Reply:
> Is that underscore URL stable and guaranteed to work? And not the side effect of frameworks that they use?

I got it from [VS Code](https://github.com/iterative/vscode-dvc/blob/main/extension/src/setup/webview/messages.ts#L142), who is using it to direct users to the token. @iterative/studio can answer whether it's stable I guess.

> Note that above the url does not handle if you are logged out

You mean it doesn't redirect after login?

Reply:
> Is that underscore URL stable and guaranteed to work? And not the side effect of frameworks that they use?

It is stable, we were using it for a couple of years already (both backend and frontend knows about it and uses it). It may be considered a side-effect of the framework we use, but we made a decision to use this and don't plan to move away from both the framework and this concept, atm.

So, you may consider it as “guaranteed”.

Reply:
Thankyou @mvshmakov for confirmation. One more question, is it possible to redirect after login?

Reply:
> One more question, is it possible to redirect after login?

Yes, with conventional `?next={url}`. Better to double-check before writing down.

Reply:
I'm not sure what you mean. Is this a question about whether it is possible to make this link work even if the user is logged out? E.g., if the user is not logged in, to redirect the user to the login page and after a login redirect to the URL which was passed before?

We have to consider a few edge cases here, and now it is not supported. If this is what you wanted, please create an issue in the Studio repo. If not, could you please elaborate on the question a bit more?

Reply:
> Is this a question about whether it is possible to make this link work even if the user is logged out? E.g., if the user is not logged in, to redirect the user to the login page and after a login redirect to the URL which was passed before?

yes, that's what I meant. Will create an issue. Thanks.